bugtool/cmd/root.go

```diff
@@ -65,26 +66,42 @@ var (
 )
 
 func init() {
+	BugtoolRootCmd.Flags().BoolVar(&archive, "archive", true, "Create archive")
```

Comment:
Should this also state that when archive == false the directory is not deleted?

Reply:
Fixed